src/transformers/modeling_encoder_decoder.py

```diff
@@ -69,10 +69,6 @@
             :meth:`transformers.PreTrainedTokenizer.__call__` for details.
 
             `What are input IDs? <../glossary.html#input-ids>`__
-        inputs_embeds (:obj:`torch.FloatTensor` of shape :obj:`(batch_size, sequence_length, hidden_size)`, `optional`):
-            Optionally, instead of passing :obj:`input_ids` you can choose to directly pass an embedded representation.
-            This is useful if you want more control over how to convert :obj:`input_ids` indices into associated
-            vectors than the model's internal embedding lookup matrix.
```

Comment:
No more support for `inputs_embeds` in encoder-decoders?

Reply:
I thought there were useless, so I just deleted them

Reply:
No just kidding! 
The `EncoderDecoderModel.forward()` can essentially accept all kind of arguments and since `inputs_embeds` and `decoder_inputs_embeds` are more optional args they could just be put into the **kwargs and would then be handed over to the correct encoder / decdoer model. But I think it's actually better to leave it as it was since every model should have `inputs_embeds` defined - will revert it :-)